app/src/main/res/layout-land/profile_chooser_profile_view.xml

```diff
@@ -39,8 +39,8 @@
 
       <de.hdodenhof.circleimageview.CircleImageView
         android:id="@+id/profile_avatar_img"
-        android:layout_width="72dp"
-        android:layout_height="72dp"
+        android:layout_width="@dimen/profile_chooser_profile_view_circular_image_size"
```

Comment:
It seems this is always corresponding to the profile avatar image. Perhaps: ``profile_chooser_avatar_circular_image_size`` might be clearer?

Reply:
Done.